tests/ConfigTest.php

```diff
@@ -12,73 +12,77 @@
  */
 class ConfigTest extends TestCase
 {
-  
+
   public function testDefineHappy() {
     Config::define('WP_SCRIPT_DEBUG', true);
     $this->assertEquals(true, Config::get('WP_SCRIPT_DEBUG'));
   }
-  
+
   public function testDefineSad() {
     $this->expectException(ConstantAlreadyDefinedException::class);
-  
+
     define('WP_SCRIPT_DEBUG', false);
     Config::define('WP_SCRIPT_DEBUG', true);
   }
-  
+
   public function testApplyHappy()
   {
     Config::define('WP_SCRIPT_DEBUG', true);
     Config::apply();
     $this->assertTrue(WP_SCRIPT_DEBUG);
   }
-  
+
   public function testApplyRedefine()
   {
     $this->expectException(ConstantAlreadyDefinedException::class);
-    
+
     Config::define('WP_SCRIPT_DEBUG', true);
     define('WP_SCRIPT_DEBUG', false);
     Config::apply();
   }
-  
+
   public function testReapply()
   {
     Config::define('WP_SCRIPT_DEBUG', true);
     Config::apply();
     Config::define('AUSTIN', 'PRO');
     Config::apply();
-    
+
     $this->assertTrue(WP_SCRIPT_DEBUG);
   }
-  
+
   /* Below are not used in Bedrock */
-  
+
   public function testGetHappy()
   {
     Config::define('WP_SCRIPT_DEBUG', true);
     Config::define('AUSTIN', 'PRO');
     Config::define('CRAIG', -1);
     Config::define('KALEN', 1);
-    
+
     $this->assertEquals(true, Config::get('WP_SCRIPT_DEBUG'));
     $this->assertEquals('PRO', Config::get('AUSTIN'));
     $this->assertEquals(-1, Config::get('CRAIG'));
     $this->assertEquals(1, Config::get('KALEN'));
+    $this->assertEquals(1, Config::get('BRANDON', 1));
```

Comment:
```suggestion
    $this->assertEquals(1, Config::get('BRANDON', 1));
    $this->assertEquals(0, Config::get('BRANDON', 0));
```

![image](https://user-images.githubusercontent.com/2192970/79187162-2fb92d80-7de1-11ea-82f0-7f5511e9dcef.png)